docs/apache-airflow/concepts/dags.rst

```diff
@@ -177,6 +179,20 @@ In much the same way a DAG instantiates into a DAG Run every time it's run,
 Tasks specified inside a DAG are also instantiated into
 :ref:`Task Instances <concepts:task-instances>` along with it.
 
+When the DAG run starts, it will have the start date and end date. This period
+describes the time when the DAG run actually 'ran.' Aside from the DAG
```

Comment:
The DAG run does not have an end date until after it _finishes_, not when it starts. So the wording here probably needs to be tweaked slightly to avoid confusion.

Reply:
sure, makes sense - will apply the suggestion accordingly.